fixtures/gtfs2ntfs/platforms/input/stops.txt

```diff
@@ -0,0 +1,16 @@
+stop_id,stop_name,stop_lat,stop_lon,location_type,parent_station,platform_code
```

Comment:
it's a big fixture just for a platform code

Reply:
@datanel  
@woshilapin told me the same thing. I take the time to reduce it?

Reply:
I just did it in 7f6aaf790546bdf5715101239c407dff176ba397.

Reply:
Nice :+1: